### PR TITLE
Allow chunk loaders to be placed upside-down

### PR DESCRIPTION
**Pull request by 61352151511**
Allows chunk loaders to be placed upside-down by placing them while looking at the bottom of another block.

Before change:

After change:

### CODE REVIEW

File: src/main/java/codechicken/chunkloader/block/BlockChunkLoader.java [hunk 1]
```diff
@@ -147,10 +156,9 @@ public int damageDropped(IBlockState state) {
         return getMetaFromState(state);
     }
 
-    @Override
-    public int getMetaFromState(IBlockState state) {
+	@Override public int getMetaFromState(IBlockState state) {
```
Comment (covers1624) [inline src/main/java/codechicken/chunkloader/block/BlockChunkLoader.java]: Override on line above.

File: src/main/java/codechicken/chunkloader/client/TileChunkLoaderRenderer.java [hunk 2]
```diff
@@ -79,7 +80,9 @@ public void renderTileEntityAt(TileChunkLoaderBase tile, double x, double y, dou
         }
         rot = ClientUtils.getRenderTime() * active / 3F;
 
-        Matrix4 pearlMat = RenderUtils.getMatrix(new Vector3(x + 0.5, y + height + (updown + 0.3) * active, z + 0.5), new Rotation(rot, new Vector3(0, 1, 0)), size);
+        double yHeight = y + height + (updown + 0.3) * active;
+        if (tile.getWorld().getBlockState(tile.getPos()).getValue(BlockChunkLoader.TYPE).isDown()) yHeight = y - (height / 2) - (updown - 0.3) * active;
```
Comment (covers1624) [inline src/main/java/codechicken/chunkloader/client/TileChunkLoaderRenderer.java]: Make this a full if statement for readability.

**Comment by covers1624** [comment]
Aside form the comments, looks good, fix those up and also make sure your formatting is the same, and it will be pulled.

**Comment by 61352151511** [comment]
Fixed those up, the formatting should be the same, if you see any changes that have messed the formatting up let me know. The lasers act the exact same as if it was facing upward (It goes up and to the center of the chunk), not sure where the rotate call would go or if it would be easier to just make it start drawing lower when upside down.

**Comment by covers1624** [comment]
Hmm, right, it should be fine then don't worry about it/

**Pull request merged**